js/onoffice-captchacontrol.js

```diff
@@ -20,4 +20,21 @@ var onOffice = onOffice || {};
 
 		return iePosition > 0 || !!navigator.userAgent.match(/Trident.*rv\:11\./);
 	};
-})();
\ No newline at end of file
+})();
+
+var requiresCaptchaForm = requiresCaptchaForm || false;
+jQuery(document).ready(function ($) {
+	if (requiresCaptchaForm == true) {
+		function addRecaptchaScript() {
+			if (!$("#recaptcha-script").length) {
+				$("<script>")
+					.attr("src", "https://www.google.com/recaptcha/api.js")
+					.attr("async", false)
+					.attr("id", "recaptcha-script")
+					.appendTo("head");
+			}
+		}
+		const targetedFormElements = "#onoffice-form input, #onoffice-form textarea, #onoffice-form button, #onoffice-form";
```

Comment:
Please use also .oo-forms as selector parent. See onoffice-honeypot.js for reference.